app/controllers/api/v1/articles_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Api
+  module V1
+    class ArticlesController < ApplicationController
+      skip_before_action :authenticate_user!, only: %i[index show]
+
+      # def index
+      #
+      # end
+
+      def show
+        article = Article.find(params[:id])
+        render json: article, serializer: ArticleSerializer
+      end
+
+      def create
+        article = ArticleService.new.create(create_params)
+        render json: article, serializer: ArticleSerializer
+      end
+
+      def destroy
+        article = Article.find(params[:id]).destroy!
```

Comment:
How about taking this code to the service?

The reason I made you write service is for easier migration to GraphQL.

Reply:
oh noice